iOS/LocalPackages/DataBrokerProtection-iOS/Sources/DataBrokerProtection-iOS/ManagingAndCoordinating/DataBrokerProtectionIOSManager.swift

```diff
@@ -30,150 +30,98 @@ import WebKit
 import BackgroundTasks
 import SwiftUI
 
-public class DefaultOperationEventsHandler: EventMapping<JobEvent> {
+/*
+ This class functions as the main coordinator for DBP on iOS (and hence the main decision maker).
+ It's the sole public inferface, and any access to DBP the main app needs should go through this.
+ It should do so using protocols, see TD for details:
+ https://app.asana.com/1/137249556945/project/481882893211075/task/1210773744858892?focus=true
+ */
 
-    public init() {
-        super.init { event, _, _, _ in
-            switch event {
-            default:
-                print("event happened")
-            }
-        }
+public class DBPIOSInterface {
```

Comment:
In creating this, it strikes me that it may be possible to go a step further and have the interface be defined in a completely separate package. This would make its proper usage a lot easier to enforce